### PR TITLE
Add short introductions to each sections in UG

### LINKED ISSUE #301: More description for each section

### LINKED ISSUE #303: Create a heading for Table of Contents

### PR DESCRIPTION
**Pull request by luo-git**
Also adds a heading for table of contents

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -334,25 +340,27 @@ Examples :
...
+drag each view to the size desired.
 ![Fig. 3. Resizing taskbar](images/screenshots/taskbar_resize.png)
 
 ![Fig. 4. Resizing filebar](images/screenshots/filebar_resize.png)
 
 ### Persistent customisation
-All of your customisation options are saved and persists even when you close the application. HelloFile remembers your choices
- so you can spend your time on more productive pursuits.
+All of your customisation options are saved and persist even when you close the application. HelloFile remembers your choices,
+so you can spend your time on more productive pursuits.
```
Comment (Ma-Yueran) [inline docs/UserGuide.md:363]: Actually I think it's not all saved, only the window size and the theme is saved, but the sizes of each panel are going to be reset to default every time.

**Review by Ma-Yueran — COMMENTED** [review]
Mostly looks good, but I have one question about the customisation part

File: docs/UserGuide.md (hunk 1 above)
Comment (luo-git) [reply docs/UserGuide.md:363]: Ok

**Review by Ma-Yueran — APPROVED** [review]
Looks good to merge!

**Pull request merged**